webapps/density-custom-chart/style.css

```
@@ -0,0 +1,100 @@
+@import '/static/public/styles/1.0.0/dku-styles.css';
+
+#spinner {
+    display: none;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    z-index: 3000;
+    width: 150px;
+    height: 150px;
+    margin: -75px 0 0 -75px;
+    border: 16px solid #f3f3f3;
+    border-radius: 50%;
+    border-top: 16px solid #3498db;
+    -webkit-animation: spin 2s linear infinite;
+    animation: spin 2s linear infinite;
+}
+
+@-webkit-keyframes spin {
+    0% { -webkit-transform: rotate(0deg); }
+    100% { -webkit-transform: rotate(360deg); }
+}
+
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
+
+.user-message-style {
+    z-index: 1500;
+    background-color: #28a9dd;
+    position: absolute;
+    top: 0px;
+    left: -1px;
+    margin-top: 10px;
+    width: 100%;
+    text-align: center;
+    padding: 10px;
+    font-family: 'SourceSansPro';
+    font-weight: 500;
+    font-size: 14px;
+    color: #ffffff;
+    box-sizing: border-box;
+}
+
+.error-message-style {
+    z-index: 1500;
+    background-color: #28a9dd;
+    position: absolute;
+    top: 0px;
+    left: -1px;
+    margin-top: 10px;
+    width: 100%;
+    text-align: center;
+    padding: 10px;
+    font-family: 'SourceSansPro';
+    font-weight: 500;
+    font-size: 14px;
+    color: #ffffff;
+    box-sizing: border-box;
+}
+
+.error-warning-view-style {
+    z-index: 1500;
+    position:absolute;
+    top:0; bottom:0; left:0; right:0;
+    display:block;
+    background:transparent;
+}
+
+.warning-view-style {
+    z-index: 1500;
+    position:absolute;
+    top:0; bottom:0; left:0; right:0;
+    display:block;
+    background:transparent;
+}
+
+.geo-chart-style {
+
+}
+
+.tooltip {
+    position: absolute;
+    visibility: hidden;
+    z-index: 1000;
+    text-align: left;
+    background-color: rgba(256, 256, 256, 1);
+    border-radius: 12px;
+    border-top-left-radius: 12px;
+    border-top-right-radius: 12px;
+    border-bottom-left-radius: 12px;
+    border-bottom-right-radius: 12px;
+    padding-left: 15px;
+    padding-right: 15px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    font-size: 12px;
+
```

Give me some Shadow there, love shadow 🤩
```suggestion
box-shadow: 0 3px 14px rgb(0 0 0 / 40%);
```

---

Removed tooltip style unless needed, it seems that introduction of leaflet tooltip make this obsolete